crates/autopilot/src/database.rs

```diff
@@ -32,7 +34,7 @@ struct Metrics {
     table_rows: prometheus::IntGaugeVec,
 
     /// Timing of db queries.
-    #[metric(labels("type"))]
+    #[metric(name = "autopilot_database_queries", labels("type"))]
```

Comment:
This metric would have the same name as the one in the orderbook crate which would then error when registering with the global prometheus registry during e2e tests.